galley/pkg/config/analysis/diag/level.go

```diff
@@ -68,3 +69,19 @@ func GetUppercaseStringToLevelMap() map[string]Level {
 	}
 	return m
 }
+
+// Type satisfies interface pflag.Value
+func (l *Level) Type() string {
```

Comment:
>> The MessageThreshold struct is one of the codes that I kept as was, but this struct is only used by istioctl analyze for now so yeah I could merge it with Level.

Ok, now I get what you meant by that--you mean the private struct that was in analyze.go. I'll address this more in the review comment.